cmd/compute-domain-daemon/computedomain.go

```diff
@@ -44,8 +47,8 @@ type ComputeDomainManager struct {
 	factory  nvinformers.SharedInformerFactory
 	informer cache.SharedIndexInformer
 
-	nodes     []*nvapi.ComputeDomainNode
-	nodesChan chan struct{}
+	previousIPs IPSet
+	nodesChan   chan []*nvapi.ComputeDomainNode
```

Comment:
lets go back to the original constructs here and pull the IPs out of `nodes` on demand before updating.

Reply:
done